scripts/deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# deploys to a remote VM
+set -e
+
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 ENVIRONMENT_NAME"
+    exit 1
+fi
+
+environment_name="$1"
+hostname="${environment_name}.libero.pub"
+scp scripts/deploy.sh "$hostname":/tmp/remote-deploy.sh
+ssh "$hostname" REVISION_BROWSER="$(scripts/latest-revision.sh git@github.com:libero/browser)" /tmp/remote-deploy.sh
```

Comment:
> REVISION_BROWSER="$(scripts/latest-revision.sh git@github.com:libero/browser)"

Is this running locally or remotely? I would say locally and the result becomes part of the command ssh runs.

Reply:
Todo: extract intermediate variable for `$revision_browser`